plugins/jobs/girder_plugin_jobs/models/job.py

```diff
@@ -585,3 +587,6 @@ def listChildJobs(self, job):
                                                 user=user,
                                                 level=AccessType.READ):
             yield r
+
+
+girder_model(Job, 'job', 'jobs')
```

Comment:
This is a hacky method to register a model.  This will be replaced with a cleaner implementation.